Woodog/templates/woodogdata/sign-up.html

```diff
@@ -0,0 +1,123 @@
+
+
+{% extends 'base.html' %}
+
+{% block content %}
+{% load static %}
+<style>
+    .container{
+        margin-top:120px;
+        margin-bottom:140px;
+    }
+    .normal-text{
+        font-family: 'Nunito', sans-serif; 
+        font-size:24px;
+        text-align: center;
+        color: #FAFAFACC;
+    }
+    .input-field{
+        margin-top:20px;
+        height: 58px;
+        margin-bottom:40px;
+        border: 1px solid #5A40AB;
+        box-sizing: border-box;
+        border-radius: 8px;
+        background-color: #1F163B;
+        color:white;
+        width:535px;
+        margin-left: auto;
+        margin-right: auto;
+    }
+    .heading{
+       font-family: 'Poppins', sans-serif;
+       color:white;
+       font-size: 36px;
+       line-height: 54px;
+       text-align: center;
+    }
+    .label{
+        color: #FAFAFA;
+        font-size: 18px;
+        margin-left:290px;
+        font-family: 'Nunito', sans-serif;
+    }
+    .account-btn{
+        margin-top: 59px;
+        background-color:  #5A40AB;
+        align-items: center;
+        padding: 16px 45px;
+        width: 535px;
+        font-size: 18px;
+        color: #FFFFFF;
+        height: 57px;
+        margin-left:290px;
+        margin-bottom:10px;
+    }
+    #btn-text{
+        color: #9A88D3;
+        margin-left:290px;
+        font-size: 18px;
+        text-decoration: none;
+       
+    }
+    @media screen and (max-device-width: 375px) {
+
+        .account-btn{
+            margin-left:0px;
+            width:100%;
+        }
+        #btn-text{
+            margin-left: 0px;
+        }
+        .label{
+            margin-left: 0px;
+        }
+        .input-field{
+            width:100%;
+        }
+    }
+
+    @media screen and (max-device-width: 1000px) {
+
+        .account-btn{
+            margin-left:0px;
+            width:100%;
+        }
+        .input-field{
+            width:100%;
+        }
+        #btn-text{
+            margin-left: 0px;
+        }
+        .label{
+            margin-left: 0px;
+        }
+    }
+</style>
+
+<div class="container col-lg-12" >
+    <p class="normal-text">Join Woodog</p>
+    <p class="heading">Create your Account</p>
+    <form style="margin-top:80px;">
+        <div class="form-group" >
+          <label for="email" class="label">Email</label>
+          <input type="email" class="form-control input-field" id="email" aria-describedby="emailHelp" placeholder="demo@space.com">
```

Comment:
Please add the `name` attribute to all the inputs.

You can refer to the existing `templates/authentication/register.html`.